### DISCUSSION TITLE
Sprint 0 review y Sprint 1 planning 2/10

### DISCUSSION THREAD

**Discussion by rpgrca** [rpgrca/proyecto-ispc-fullstack#58] [General]
Ya que no nos podemos juntar mañana 3/10 hoy hicimos la sprint 0 review y la sprint 1 planning. Hemos terminado virtualmente todo lo que teníamos para hacer para el sprint 0, quedan transformar las últimas historias de usuario a issues y reflejarlo en el archivo de requisitos que se harán durante el día.

Con respecto al nuevo sprint ya que tiene una duración de 2 semanas decidimos que para la próxima reunión semanal del domingo 9 vamos a tener terminado todo lo de HTML, es decir, tener un sitio que se pueda navegar aunque no necesariamente con estilo (que se irá agregando a medida que se termine la maquetación). Como al parecer necesitamos usar Bootstrap vamos a realizar un estilo sobrio pero minimalista para no realizar un estilo grandioso que luego sea difícil de cambiar. También hicimos una demostración de cómo habría que transformar los issues a una rama de desarrollo, como hacer un fetch y checkout de esa rama, luego como subir esa rama y crear un pull request. También hicimos una demostración de cómo editar una historia de usuario para agregar más tareas a las ya existentes.

También vamos a configurar el repositorio para intentar que nadie pueda subir a main directamente sino que todo tenga que pasar por dev, y también para que los pull request pasen por un code review que verifique el código.

Cada miembro tomará las tareas que pueda realizar y se enfocará en la maquetación, una vez terminada decidirá si quiere trabajar en otra maquetación o agregar algo de estilo, sin embargo la prioridad es terminar todas las maquetaciones.

**Comment by claubott** [comment]
Muy bueno el encuentro de hoy! Ya me autoasigné la página de contacto.

**Comment by NataliaAlvarezIspc** [comment]
Gracias Roberto!
Ya están agregadas las historias de usuarios y las tareas.

**Comment by AhumadaCeleste** [comment]
Excelente. Gracias Roberto. Hoy mismo comienzo a trabajar con las tareas asignadas, par poder contar con el tiempo necesario y consultar en caso de precisar ayuda.
